runtime/src/bank.rs

```diff
@@ -2747,6 +2747,40 @@ impl Bank {
             .unwrap_or(default))
     }
 
+    /// Get all pyth related pubkeys from environment variables
+    /// or return default if the variable is not set.
+    pub fn get_pyth_keys(
```

Comment:
I would rename this to `get_accumulator_keys`

Reply:
fixed thanks